tests/test_dbos.py

```diff
@@ -1477,3 +1487,70 @@ def test_workflow(x: int) -> int:
 
     # Clean up the environment variable
     del os.environ["DBOS__VMID"]
+
+
+def test_workflow_timeout(dbos: DBOS) -> None:
```

Comment:
suggest to add a test where the parent exits early and the children still timeout

Reply:
Done